tests/admin_scripts/tests.py

```diff
@@ -1403,6 +1403,8 @@ def test_empty_allowed_hosts_error(self):
 
 
 class ManageTestserver(AdminScriptTestCase):
+    from django.core.management.commands.loaddata import Command as LoaddataCommand
+    from django.core.management.commands.runserver import Command as RunserverCommand
     from django.core.management.commands.testserver import Command as TestserverCommand
```

Comment:
Is there any reason for these class level imports 🤔 ?

Reply:
I think it's unnecessary.